org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaLanguageServerPlugin.java

```diff
@@ -619,4 +622,11 @@ public synchronized static ExecutorService getExecutorService() {
 		}
 		return pluginInstance.executorService;
 	}
+
+	public synchronized static ICompletionContributionService getCompletionRankingService() {
+		if (pluginInstance.completionRankingService == null) {
+			pluginInstance.completionRankingService = new CompletionRankingService();
```

Comment:
Since you have renamed the interface name, it might be better to rename the impl class as `CompletionContributionService`, and the method `getCompletionRankingService` to `getCompletionContributionService`.

Reply:
Thanks to point out!